File: estructura_datos/cadenas_caracteres/ejemploV1.py
def checkPal(str):
    transformedStr = str.lower().replace(" ", "")
    transformedStrReversed = transformedStr[::-1]
    return transformedStr == transformedStrReversed


def run():
    print(checkPal("allivessevilla"))


if __name__ == "__main__":
    run()

File: estructura_datos/cadenas_caracteres/ejemploV1.js
const checkPal = (str) => {
  const transformedStr = str.toLowerCase().replace(/\s/g, "");
  const transformedStrReverted = transformedStr.split("").reverse().join("");
  return transformedStr === transformedStrReverted;
};

console.log(checkPal("allivessevilla"));
